mongodb-aggregate-query-support-test/src/main/java/com/github/krr/mongodb/embeddedmongo/config/LinuxPackageIoUtilImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.github.krr.mongodb.embeddedmongo.config;
+
+import com.github.krr.mongodb.embeddedmongo.config.impl.LinuxPackageIoUtil;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+public class LinuxPackageIoUtilImpl implements LinuxPackageIoUtil {
+  @Override
+  public boolean isExists(File file) {
+    return file.exists();
+  }
+
+  @Override
+  public List<String> readFile(File file) {
+    List<String> fileContents = new ArrayList<>();
+    try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+      while (bufferedReader.ready()) {
+        fileContents.add(bufferedReader.readLine());
+      }
+    }
+    catch (IOException e) {
+      log.error("Error reading the file {}", file.getName(), e);
```

Comment:
Do we want to return from this exception normally?  You've effectively swallowed the exception.

Reply:
Here the runtime exception will still be thrown
In case of IO exception, the distribution and version fields would remain “”, which will be caught later
The Runtime Exception will be caught later where there is an empty string check for directory and version.